File: apis/src/main/resources/META-INF/dirigible/custom-api/ExampleResponse.ts
import { Example } from "./Example";

export class ExampleResponse {

    private examples: Example[] = [];

    public getExamples(): Example[] {
        return this.examples;
    }

    public setExamples(examples: Example[]): void {
        this.examples = examples;
    }

    public withExamples(examples: Example[]): ExampleResponse {
        this.setExamples(examples);
        return this;
    }
}

File: apis/src/main/java/io/dirigible/samples/api/domain/output/ExampleResponse.java
package io.dirigible.samples.api.domain.output;

import java.util.ArrayList;
import java.util.List;

import io.dirigible.samples.api.domain.Example;

public class ExampleResponse {

	private List<Example> examples = new ArrayList<>();

	public List<Example> getExamples() {
		return examples;
	}

	public void setExamples(List<Example> examples) {
		this.examples = examples;
	}

	public ExampleResponse withExamples(List<Example> examples) {
		setExamples(examples);
		return this;
	}
}
